src/append/rolling_file/policy/compound/trigger/time.rs

```diff
@@ -23,27 +23,27 @@ use crate::config::{Deserialize, Deserializers};
 #[derive(Copy, Clone, Eq, PartialEq, Hash, Debug, Default, serde::Deserialize)]
 #[serde(deny_unknown_fields)]
 pub struct TimeTriggerConfig {
-    interval: TimeTriggerInterval,
+    /// The date/time interval between log file rolls.
+    pub interval: TimeTriggerInterval,
```

Comment:
Why the decision to make everything pub? This doesn't align with the other triggers